modules/core/04-channel/types/msgs_test.go

```diff
@@ -350,14 +351,14 @@ func (suite *TypesTestSuite) TestMsgChannelCloseConfirmValidateBasic() {
 		msg     *types.MsgChannelCloseConfirm
 		expPass bool
 	}{
-		{"", types.NewMsgChannelCloseConfirm(portid, chanid, suite.proof, height, addr), true},
-		{"too short port id", types.NewMsgChannelCloseConfirm(invalidShortPort, chanid, suite.proof, height, addr), false},
-		{"too long port id", types.NewMsgChannelCloseConfirm(invalidLongPort, chanid, suite.proof, height, addr), false},
-		{"port id contains non-alpha", types.NewMsgChannelCloseConfirm(invalidPort, chanid, suite.proof, height, addr), false},
-		{"too short channel id", types.NewMsgChannelCloseConfirm(portid, invalidShortChannel, suite.proof, height, addr), false},
-		{"too long channel id", types.NewMsgChannelCloseConfirm(portid, invalidLongChannel, suite.proof, height, addr), false},
-		{"channel id contains non-alpha", types.NewMsgChannelCloseConfirm(portid, invalidChannel, suite.proof, height, addr), false},
-		{"empty proof", types.NewMsgChannelCloseConfirm(portid, chanid, emptyProof, height, addr), false},
+		{"", types.NewMsgChannelCloseConfirm(portid, chanid, suite.proof, height, addr, 0), true},
```

Comment:
if there's a constant for the counteparty upgrade sequence, you could use that instead of 0, but I'm perfectly fine with this. Feel free to resolve